From: Eric Dorland <eric@debian.org>
Date: Mon, 3 Jul 2017 17:15:34 -0400
Subject: Drop @SHELL@

Using @SHELL@ is a reproducibility problem and shouldn't be necessary on
Debian systems.
---
 bin/automake.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bin/automake.in b/bin/automake.in
index d356336..b917d32 100644
--- a/bin/automake.in
+++ b/bin/automake.in
@@ -42,7 +42,7 @@ BEGIN
   # it turns up other systems need the same thing.  After all,
   # if SHELL is used, ./configure's SHELL is always better than
   # the user's SHELL (which may be something like tcsh).
-  $ENV{'SHELL'} = '@SHELL@' if exists $ENV{'DJDIR'};
+  $ENV{'SHELL'} = '/bin/sh' if exists $ENV{'DJDIR'};
 }
 
 use Automake::Config;
